api/db/changelog/db.changelog-13-blockscore-id.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="dmohs" id="changelog-13-blockscore-id">
+    <addColumn tableName="user">
+      <column name="blockscore_id" type="varchar(30)">
```

Comment:
Did you find out whether this identifier (or any other identifier) in Blockscore is the same for all calls to createPerson with the same parameters?

If not, we're probably going to need to have some fingerprint tracking uniqueness of people... like storing a hash of their name, DOB, identifiers, etc...

And we may also need a timestamp to track when a user got themselves verified. That way we can look for the existence of an earlier user row with the same fingerprint.

(This is all assuming that we actually want to prevent a single person from signing up for multiple accounts -- we should confirm.)

Fine if you don't want to tackle it now, but we should add a JIRA ticket and TODO.

Reply:
https://precisionmedicineinitiative.atlassian.net/browse/RW-291